examples/betterauth/src/app/(app)/page.tsx

```diff
@@ -12,7 +12,7 @@ import {
 import Image from "next/image";
 
 export default function Home() {
-  const me = useAccount(Account, { resolve: { profile: {} } });
```

Comment:
I don't like this much, especially because `co.account()` will return a different schema on each render, resetting the subscrition.

We could:
1. Make `co.account()` return a cached schema (this should be easy)
2. Change the examples to export a JazzAccount from the schema, even if they don't use the root